packages/app/src/runner/event-manager.ts

```diff
@@ -235,6 +235,10 @@ export class EventManager {
       this.saveState(state)
     })
 
+    this.reporterBus.on('debug:dismiss', () => {
+      this.localBus.emit('debug:dismiss')
+    })
```

Comment:
Also there is an ongoing effort to type these, might be good to add it here https://github.com/cypress-io/cypress/blob/develop/packages/app/src/runner/event-manager-types.ts#L34

Reply:
Done in [9074f63](https://github.com/cypress-io/cypress/pull/25265/commits/9074f631a26c33d0f83b298f7ed938dfa8653e3e)